src/outagesim/backup_reliability.jl

```diff
@@ -91,14 +172,51 @@ julia> starting_probabilities(2, 0.99, 0.05)
  0.00354025  0.11192  0.88454
 ```
 """
-function starting_probabilities(N::Int, OA::Real, failure_to_start::Real)::Matrix{Float64} 
-    M = markov_matrix(N, (1-OA) + failure_to_start*OA) 
-    G = hcat(zeros(1, N), 1)
+function starting_probabilities(num_gen::Int, gen_operational_availability::Real, gen_failure_to_start::Real)::Matrix{Float64} 
+    M = markov_matrix(num_gen, (1-gen_operational_availability) + gen_failure_to_start*gen_operational_availability) 
```

Comment:
Not ideal to calculate this whole matrix just to take the last row

Reply:
This may not be important to address if solve times are already good.

Reply:
Updated function per recommendation

Reply:
@sericson0 I'm not seeing how my comment was addressed

Reply:
From a conversation we had I thought the comment was to add the [end, :] to the Markov matrix instead of the original way of calculating M, then multiplying by G to get the end row? Apologies if I misunderstood. 

Or were you thinking of doing the calculation without creating the Markov matrix to begin with? May be slightly better but we already have the function for making the Markov matrix, the matrix is pretty small and does not take much time to create, and only gets made once.

Reply:
Agreed not that important. Resolving.